library/src/main/java/com/leinardi/android/speeddial/FabWithLabelView.java

```diff
@@ -344,7 +349,33 @@ private void setLabelColor(@ColorInt int color) {
     }
 
     private void setLabelBackgroundColor(@ColorInt int color) {
-        mLabelCardView.setCardBackgroundColor(ColorStateList.valueOf(color));
+        if (color == Color.TRANSPARENT) {
+            mLabelCardView.setCardBackgroundColor(Color.TRANSPARENT);
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+                mLabelCardViewElevation = mLabelCardView.getElevation();
+                mLabelCardView.setElevation(0);
+            } else {
+                mLabelCardView.setBackgroundColor(Color.TRANSPARENT);
+                mLabelCardViewBackground = mLabelCardView.getBackground();
+            }
+        } else {
+            mLabelCardView.setCardBackgroundColor(ColorStateList.valueOf(color));
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+                if (mLabelCardViewElevation != 0) {
+                    mLabelCardView.setElevation(mLabelCardViewElevation);
+                    mLabelCardViewElevation = 0;
+                }
+            } else {
+                if (mLabelCardViewBackground != null) {
+                    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN) {
+                        mLabelCardView.setBackground(mLabelCardViewBackground);
+                    } else {
+                        mLabelCardView.setBackgroundDrawable(mLabelCardViewBackground);
```

Comment:
**Reporter**: Error Prone
**Rule**: deprecation
**Severity**: WARN
**File**: library/src/main/java/com/leinardi/android/speeddial/FabWithLabelView.java L373

setBackgroundDrawable(Drawable) in View has been deprecated

mLabelCardView.setBackgroundDrawable(mLabelCardViewBackground);^

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<1965684139>*